instanceidhandler/v1/instanceidhandler.go

```diff
@@ -6,7 +6,28 @@ import (
 	"fmt"
 
 	"github.com/kubescape/k8s-interface/k8sinterface"
-	"github.com/kubescape/k8s-interface/workloadinterface"
+)
+
+// labels
+const (
+	labelPrefix                 = "kubescape.io"
+	labelFormatKeyApiGroup      = labelPrefix + "/workload-api-group"
```

Comment:
why do we stop exporting them?

Reply:
Since we have a function for `GetLabels`, there is no need for exposing the labels. (the labels are not meant for internal use).
Do you have a different thought in mind?

Reply:
I think we're using labels to let us or users select on them... So maybe it's better to have them abstracted somehow to facilitate their reuse. Otherwise we could just use annotations if we want this info to be internal